docs/platform/developing/flashing-rpi.md

```diff
@@ -0,0 +1,511 @@
+# Flashing an Image to RPI
+
+## Introduction
+
+This topic describes how to flash Tizen on SD card with or without IoT Setup Manager and setting up Raspberry Pi 3 or 4.
+
+## Supported systems
+
+|Division	|Ubuntu	|Windows	|
+|:---:		|:---:	|:---:		|
+|Headless 32-bit|Yes	|No			|
+|Headed 32-bit|Yes	|Yes		|
+|Headed 64-bit|Yes	|No			|
+
+> **Note**
+>
+> In windows, only EXT4 type file of rootfs.img is supported to flash rpm bundles. (While using Headed 64 bit image and Headless 32bit image, RPMs cannot be installed through IOT setup manager because rootsfs.img is of BTRFS type file.)
+
+## Prerequisites
+
+### Operating System
+
+IoT Setup Manager supports the following operating systems:
+
+-   Windows 7 and later (64 and 32 bit)
+-   Ubuntu 14.04 LTS and later (64 and 32 bit)
+
+### Ubuntu Tools
+
+If you are using Ubuntu, the following packages must be installed in addition to the IoT Setup Manager:
+
+-   rpm2cpio
+-   cpio
+
+If the packages are not installed, run the following command to install a package:
+
+```
+sudo apt-get install <package-name>
+```
+
+### Download binaries
+
+You must have the supported binary images in your computer. You can download the binary images from:
+
+-   To Download images visit [Downloads](http://download.tizen.org/releases/milestone/tizen)
+-   Then go to 6.0-unified (or "unified" whichever is present) -> latest -> images -> standard.
+-   For different devices or profiles download the compressed file from:
```

Comment:
Please use a numbered list as it is a sequential step and please use the style guide for procedures:

Reference: https://developers.google.com/style/procedures

Reply:
Fixed